diff -uraN xscreensaver-6.02/driver/XScreenSaver.ad.in OpenMandriva-xscreensaver-6.02/driver/XScreenSaver.ad.in
--- xscreensaver-6.02/driver/XScreenSaver.ad.in	2021-10-11 20:49:37.173116852 +0200
+++ OpenMandriva-xscreensaver-6.02/driver/XScreenSaver.ad.in	2022-01-10 18:19:43.955661401 +0100
@@ -29,23 +29,23 @@
 ! /* (xrdb prevention kludge: whole file)
 
 *mode:			random
-*timeout:		0:10:00
+*timeout:		0:05:00
 *cycle:			0:10:00
-*lockTimeout:		0:00:00
+*lockTimeout:		0:10:00
 *passwdTimeout:		0:00:30
-*dpmsEnabled:		False
+*dpmsEnabled:		True
 *dpmsQuickoffEnabled:	False
-*dpmsStandby:		2:00:00
-*dpmsSuspend:		2:00:00
-*dpmsOff:		4:00:00
-*grabDesktopImages:	True
+*dpmsStandby:		0:10:00
+*dpmsSuspend:		0:15:00
+*dpmsOff:		0:20:00
+*grabDesktopImages:	False
 *grabVideoFrames:	False
 *chooseRandomImages:	@DEFAULT_IMAGES_P@
 ! This can be a local directory name, or the URL of an RSS or Atom feed.
 *imageDirectory:	@DEFAULT_IMAGE_DIRECTORY@
 *nice:			10
 *memoryLimit:		0
-*lock:			False
+*lock:			True
 *verbose:		False
 *fade:			True
 *unfade:		True
@@ -55,15 +55,19 @@
 *splashDuration:	0:00:05
 *visualID:		default
 *installColormap:	True
-*ignoreUninstalledPrograms: False
+*ignoreUninstalledPrograms: True
 *authWarningSlack:	20
 
-*textMode:		file
+*textMode:		data
 *textLiteral:		XScreenSaver
 *textFile:		@DEFAULT_TEXT_FILE@
 *textProgram:		fortune
 *textURL:		https://en.wikipedia.org/w/index.php?title=Special:NewPages&feed=rss
 
+! default mode is one screensaver
+*mode:                  one
+*selected:              0
+
 ! This is what the "Settings" button on the splash screen runs.
 *demoCommand: xscreensaver-settings
 
@@ -97,8 +101,8 @@
 !
 !   non-Gnome systems:
 !
-@NOGNOME@*loadURL: x-www-browser '%s' || firefox '%s' || chromium-browser '%s'
-@NOGNOME@*manualCommand: xterm -sb -fg black -bg gray75 -T '%s manual' \
+@NOGNOME@*loadURL: @WITH_BROWSER@ '%s'
+@NOGNOME@*manualCommand: xvt -T '%s manual' \
 @NOGNOME@		    -e /bin/sh -c 'man "%s" ; read foo'
 
 
@@ -325,6 +329,9 @@
 
 *programs:								      \
 				maze -root				    \n\
+default-n:             "OpenMandriva" chbg -xscreensaver -mode smart -bg "#000000"  \
+                                        -interval 0.16 -effect 1 -speed 500                  \
+                                        -R `/bin/ls /usr/share/mdk/screensaver/*`          \n\
 @GL_KLUDGE@ GL: 				superquadrics -root			    \n\
 				attraction -root			    \n\
 				blitspin -root				    \n\
@@ -655,6 +662,7 @@
 *hacks.morph3d.name:        Morph 3D
 *hacks.nerverot.name:       Nerve Rot
 *hacks.noseguy.name:        Nose Guy
+*hacks.openmandriva.name:   OpenMandriva
 *hacks.pacman.name:         Pac-Man
 *hacks.photopile.name:      Photo Pile
 *hacks.popsquares.name:     Pop Squares
diff -uraN xscreensaver-6.02/hacks/config/OpenMandriva.xml OpenMandriva-xscreensaver-6.02/hacks/config/OpenMandriva.xml
--- xscreensaver-6.02/hacks/config/OpenMandriva.xml	1970-01-01 01:00:00.000000000 +0100
+++ OpenMandriva-xscreensaver-6.02/hacks/config/OpenMandriva.xml	2022-01-10 18:25:54.671363599 +0100
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<screensaver name="chbg" _label="OpenMandriva">
+
+  <command arg="-xscreensaver -mode smart -bg "#000000" -interval 0.16 -effect 1 -speed 500 -R /usr/share/mdk/screensaver/"/>
+
+  <_description>
+This is the official OpenMandriva screensaver.
+Visit www.openmandriva.org
+
+Written by OpenMandriva Association; 2021.
+  </_description>
+</screensaver> 
diff -uraN xscreensaver-6.02/hacks/munge-ad.pl OpenMandriva-xscreensaver-6.02/hacks/munge-ad.pl
--- xscreensaver-6.02/hacks/munge-ad.pl	2021-02-07 09:24:43.097328233 +0100
+++ OpenMandriva-xscreensaver-6.02/hacks/munge-ad.pl	2022-01-10 18:30:03.110584813 +0100
@@ -153,6 +153,7 @@
     if (-f "$hack.c" || -f "$hack") { $glp = 0; }
     elsif (-f "glx/$hack.c") { $glp = 1; }
     elsif ($hack eq 'companioncube') { $glp = 1; }  # kludge
+    elsif ($hack eq 'OpenMandriva') { $glp = 1; }  # kludge
     elsif ($dis != 2) { error ("is $hack X or GL?"); }
 
     $counts[($disable{$hack} || 0)]++;
diff -uraN xscreensaver-6.02/po/POTFILES.in OpenMandriva-xscreensaver-6.02/po/POTFILES.in
--- xscreensaver-6.02/po/POTFILES.in	2021-10-12 03:04:31.262390680 +0200
+++ OpenMandriva-xscreensaver-6.02/po/POTFILES.in	2022-01-10 18:30:44.811257386 +0100
@@ -5,6 +5,7 @@
 driver/screensaver-properties.desktop.in
 driver/subprocs.c
 driver/xscreensaver.ui
+hacks/config/OpenMandriva.xml
 hacks/config/abstractile.xml
 hacks/config/anemone.xml
 hacks/config/anemotaxis.xml
