diff -uraN xscreensaver-6.05/driver/demo-Gtk.c omv-xscreensaver-6.05/driver/demo-Gtk.c
--- xscreensaver-6.05/driver/demo-Gtk.c	2022-09-10 02:50:09.914759315 +0200
+++ omv-xscreensaver-6.05/driver/demo-Gtk.c	2022-10-31 20:05:29.480193535 +0100
@@ -832,8 +832,13 @@
   flush_dialog_changes_and_save (s);
   xscreensaver_command (s->dpy, XA_EXIT, 0, FALSE, NULL);
   sleep (1);
+    if (getuid() == 0) {
+     if (system ("TEMPXAUTHORITY=`su - nobody -c \"/bin/mktemp /tmp/xscreensaver.XXXXXX\"` && xauth extract $TEMPXAUTHORITY $DISPLAY && XAUTHORITY=$TEMPXAUTHORITY xscreensaver --splash && rm -f $TEMPXAUTHORITY &") < 0)
+      fprintf (stderr, "%s: fork error\n", blurb());
+  } else {
   if (system ("xscreensaver --splash &") < 0)
     fprintf (stderr, "%s: fork error\n", blurb());
+    }
 
   await_xscreensaver (s);
 }
