diff -up xscreensaver-6.04/driver/XScreenSaver.ad.in.defaultconfig xscreensaver-6.04/driver/XScreenSaver.ad.in
--- xscreensaver-6.04/driver/XScreenSaver.ad.in.defaultconfig	2022-05-29 21:41:47.122482777 +0200
+++ xscreensaver-6.04/driver/XScreenSaver.ad.in	2022-06-29 12:34:22.716761649 +0200
@@ -29,23 +29,23 @@
 ! /* (xrdb prevention kludge: whole file)
 
 *mode:			random
-*timeout:		0:10:00
+*timeout:		0:05:00
 *cycle:			0:10:00
-*lockTimeout:		0:00:00
+*lockTimeout:		0:10:00
 *passwdTimeout:		0:00:30
-*dpmsEnabled:		False
+*dpmsEnabled:		True
 *dpmsQuickoffEnabled:	False
-*dpmsStandby:		2:00:00
-*dpmsSuspend:		2:00:00
-*dpmsOff:		4:00:00
-*grabDesktopImages:	True
+*dpmsStandby:		0:10:00
+*dpmsSuspend:		0:15:00
+*dpmsOff:		0:20:00
+*grabDesktopImages:	False
 *grabVideoFrames:	False
 *chooseRandomImages:	@DEFAULT_IMAGES_P@
 ! This can be a local directory name, or the URL of an RSS or Atom feed.
 *imageDirectory:	@DEFAULT_IMAGE_DIRECTORY@
 *nice:			10
 *memoryLimit:		0
-*lock:			False
+*lock:			True
 *verbose:		False
 *fade:			True
 *unfade:		True
--- xscreensaver-6.06/driver/XScreenSaver.ad.in.orig	2022-12-26 10:21:44.212299706 +0000
+++ xscreensaver-6.06/driver/XScreenSaver.ad.in	2022-12-26 10:24:33.832238256 +0000
@@ -325,6 +325,7 @@
 
 *programs:								      \
 				maze --root				    \n\
+default-n:			webcollage --root --directory /usr/share/mdk/screensaver 
 @GL_KLUDGE@ GL: 				superquadrics --root			    \n\
 				attraction --root			    \n\
 				blitspin --root				    \n\
