diff -Naur xscreensaver-5.32/driver/XScreenSaver.ad.in xscreensaver-5.32.tpg/driver/XScreenSaver.ad.in
--- xscreensaver-5.32/driver/XScreenSaver.ad.in	2014-11-18 22:54:02.000000000 +0000
+++ xscreensaver-5.32.tpg/driver/XScreenSaver.ad.in	2015-04-23 00:50:36.294782548 +0000
@@ -29,23 +29,23 @@
 ! /* (xrdb prevention kludge: whole file)
 
 *mode:			random
-*timeout:		0:10:00
+*timeout:		0:05:00
 *cycle:			0:10:00
-*lockTimeout:		0:00:00
+*lockTimeout:		0:10:00
 *passwdTimeout:		0:00:30
-*dpmsEnabled:		False
+*dpmsEnabled:		True
 *dpmsQuickoffEnabled:	False
-*dpmsStandby:		2:00:00
-*dpmsSuspend:		2:00:00
-*dpmsOff:		4:00:00
-*grabDesktopImages:	True
+*dpmsStandby:		0:10:00
+*dpmsSuspend:		0:15:00
+*dpmsOff:		0:20:00
+*grabDesktopImages:	False
 *grabVideoFrames:	False
 *chooseRandomImages:	@DEFAULT_IMAGES_P@
 ! This can be a local directory name, or the URL of an RSS or Atom feed.
 *imageDirectory:	@DEFAULT_IMAGE_DIRECTORY@
 *nice:			10
 *memoryLimit:		0
-*lock:			False
+*lock:			True
 *verbose:		False
 *timestamp:		True
 *fade:			True
@@ -56,10 +56,10 @@
 *splashDuration:	0:00:05
 *visualID:		default
 *captureStderr: 	True
-*ignoreUninstalledPrograms: False
+*ignoreUninstalledPrograms: True
 *authWarningSlack:	20
 
-*textMode:		file
+*textMode:		date
 *textLiteral:		XScreenSaver
 *textFile:		@DEFAULT_TEXT_FILE@
 *textProgram:		fortune
@@ -82,6 +82,10 @@
 ! (xscreensaver not covering the whole screen)
 GetViewPortIsFullOfLies: False
 
+! default mode is one screensaver
+*mode:                  one
+*selected:              0
+
 ! This is what the "Demo" button on the splash screen runs (/bin/sh syntax.)
 *demoCommand: xscreensaver-demo
 
@@ -100,13 +104,13 @@
 !   Gnome 2.4, 2.6: (yelp can't display man pages, as of 2.6.3)
 !
 @GNOME24@*loadURL: @WITH_BROWSER@ '%s'
-@GNOME24@*manualCommand: gnome-terminal --title '%s manual' \
+@GNOME24@*manualCommand: xvt --title '%s manual' \
 @GNOME24@		--command '/bin/sh -c "man %s; read foo"'
 !
 !   Gnome 2.2:
 !
 @GNOME22@*loadURL: gnome-url-show '%s'
-@GNOME22@*manualCommand: gnome-terminal --title '%s manual' \
+@GNOME22@*manualCommand: xvt --title '%s manual' \
 @GNOME22@		--command '/bin/sh -c "man %s; read foo"'
 !
 !   Gnome 1.4:
@@ -116,8 +120,8 @@
 !
 !   non-Gnome systems:
 !
-@NOGNOME@*loadURL: firefox '%s' || mozilla '%s' || netscape '%s'
-@NOGNOME@*manualCommand: xterm -sb -fg black -bg gray75 -T '%s manual' \
+@NOGNOME@*loadURL: @WITH_BROWSER@ '%s'
+@NOGNOME@*manualCommand: xvt -T '%s manual' \
 @NOGNOME@		    -e /bin/sh -c 'man "%s" ; read foo'
 
 
@@ -157,7 +161,9 @@
 ! screen savers interactively.
 !
 *programs:								      \
-				maze -root				    \n\
+default-n:             "OpenMandriva" chbg -xscreensaver -mode smart -bg "#000000"  \
+                         -interval 0.16 -effect 1 -speed 500                  \
+                         -R `/bin/ls /usr/share/mdk/screensaver/*`          \n\
 @GL_KLUDGE@ GL: 				superquadrics -root			    \n\
 				attraction -root			    \n\
 				blitspin -root				    \n\
@@ -487,6 +493,7 @@
 *hacks.morph3d.name:        Morph3D
 *hacks.nerverot.name:       NerveRot
 *hacks.noseguy.name:        NoseGuy
+*hacks.openmandriva.name:   OpenMandriva
 *hacks.popsquares.name:     PopSquares
 *hacks.projectiveplane.name:ProjectivePlane
 *hacks.quasicrystal.name:   QuasiCrystal
diff -Naur xscreensaver-5.32/hacks/Makefile.in xscreensaver-5.32.tpg/hacks/Makefile.in
--- xscreensaver-5.32/hacks/Makefile.in	2014-11-19 07:24:01.000000000 +0000
+++ xscreensaver-5.32.tpg/hacks/Makefile.in	2015-04-23 00:50:36.295782575 +0000
@@ -128,7 +128,7 @@
 # Programs that are mentioned in XScreenSaver.ad, and that have XML files,
 # but that are not shipped with xscreensaver itself.
 #
-EXTERNALS	= cosmos electricsheep fireflies goban \
+EXTERNALS	= cosmos electricsheep fireflies OpenMandriva goban \
 		  sphereeversion ssystem xaos xdaliclock xearth xfishtank \
 		  xmountains xplanet xsnow
 
diff -Naur xscreensaver-5.32/hacks/config/OpenMandriva.xml xscreensaver-5.32.tpg/hacks/config/OpenMandriva.xml
--- xscreensaver-5.32/hacks/config/OpenMandriva.xml	1970-01-01 00:00:00.000000000 +0000
+++ xscreensaver-5.32.tpg/hacks/config/OpenMandriva.xml	2015-04-23 00:51:28.437163482 +0000
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<screensaver name="chbg" _label="OpenMandriva">
+
+  <command arg="-xscreensaver -mode smart -bg "#000000" -interval 0.16 -effect 1 -speed 500 -R /usr/share/mdk/screensaver/"/>
+
+  <_description>
+This is the official OpenMandriva screensaver.
+Visit www.openmandriva.org
+
+Written by OpenMandriva Association; 2015.
+  </_description>
+</screensaver>
